api/models/series.go

```diff
@@ -437,3 +438,47 @@ type SeriesForPickle struct {
 	Values         []interface{} `pickle:"values" msg:"values"`
 	PathExpression string        `pickle:"pathExpression" msg:"pathExpression"`
 }
+
+// Csv formats like graphite's render&format=csv , like so: (note the null points)
+// sumSeries(carbon.agents.graphitemon-a.*),2021-05-04 11:27:41,
+// sumSeries(carbon.agents.graphitemon-a.*),2021-05-04 11:27:42,28066914.362149723
+// sumSeries(carbon.agents.graphitemon-a.*),2021-05-04 11:27:43,
+// sumSeries(carbon.agents.graphitemon-a.*),2021-05-04 11:27:44,
+func (series SeriesByTarget) Csv(buf []byte) ([]byte, error) {
+	buffer := bytes.NewBuffer(buf)
+	for _, serie := range series {
+		for _, point := range serie.Datapoints {
+
+			_, err := buffer.WriteString(serie.Target)
+			if err != nil {
+				return nil, err
+			}
+			_, err = buffer.WriteRune(',')
+			if err != nil {
+				return nil, err
+			}
+
+			time := time.Unix(int64(point.Ts), 0).UTC()
+			_, err = buffer.WriteString(time.Format("2006-01-02 15:04:05"))
```

Comment:
Is this the time format we want to use? Maybe we could also add another field or optional field which is the unix timestamp?

Or is the goal of this just to output the exact same thing that Graphite would?

Reply:
> Or is the goal of this just to output the exact same thing that Graphite would?

yep. in the future we can worry about more options